### DISCUSSION TITLE
is there a function to test for anova assumptions?

### DISCUSSION THREAD

**Discussion by microzi24** [ChiLiubio/microeco#433] [Q&A]
Hey,
I am wondering if there is a fucntion within microeco to test for anova assumptions (i.e. Levene´s test and Shapiro-WIlk' test).  Would like to test on alpha diversity indices (in trans_alpha) before running anova.

**Comment by ChiLiubio** [comment]
Hi. Thanks for the suggestion! I will try to add such test.

**Reply by ChiLiubio** [reply]
Hi. `anova_varequal_test` parameter has been added for the test on equality of variances in one-way anova. Please reinstall the package from github and try again.
```
library(microeco)
data(dataset)
t1 <- trans_alpha$new(dataset = dataset, group = "Group")
t1$cal_diff(method = "anova", anova_varequal_test = TRUE, measure = "Shannon")
t1$res_diff
```

**Reply by microzi24** [reply]
Hi, 
that is awesome, thanks! 
I´ve tried the argument after reinstalling, but it is not found (see the error):

Error in post_test_function(model, group, main = measure, alpha = alpha,  : 
  unused argument (anova_varequal_test = TRUE)

Could you have a look?

**Reply by ChiLiubio** [reply]
Hi. It seems like the new parameter is not found. Please make sure that the steps you use to reinstall the package are:
```
devtools::install_github("ChiLiubio/microeco")
```
Then after you type `?trans_alpha` , you should find the anova_varequal_test parameter in the `cal_diff` function.

**Reply by microzi24** [reply]
It works now. Apparently it was an issue with installation. Thanks!
